gdcdictionary/schemas/exposure.yaml

```diff
@@ -291,6 +328,146 @@ properties:
         $ref:
           - "_terms_enum.yaml#/tobacco/common"
 
+  occupation_duration_years:
+    type: integer
+    maximum: 2050
+    minimum: 1900
```

Comment:
maybe a dumb question: is this referring to how long in a person's life they are engaged in the occupation? or which calendar years? if the former, I think max of 89 and min of 0?

Reply:
occupation_duration_years:
  common:
    description: >-
      The number of years a patient worked in a specific occupation.
      
This is what the terms file says.  @tlicht3 do you have any thoughts?

Reply:
@wwysoc2 I think this was me working too quickly... agree with @wteouchicago that this should be 0-89.

Reply:
Fixed